doc/schemas/options_schema.json

```diff
@@ -293,6 +293,18 @@
       "title": "DebugOutput schema",
       "description": "Specifies whether Wangscape should write debug data during tile generation.",
       "default": false
+    },
+    "DebugTileResolution": {
+      "type": "array",
+      "title": "DebugTileResolution schema",
+      "description": "Specifies the resolution of the images written as debug data during tile generation.",
+      "items": {
+        "type": "integer",
+        "multipleOf": 1,
+        "minimum": 1
+      },
+      "minItems": 2,
+      "maxItems": 2
```

Comment:
Same thing here: could it be `{"width": w, "height: h}`?

I think it will be nice to stick to one resolution format to use everywhere (it could be an array, if you think it's sufficient).